ansible/roles/ubuntu/tasks/ubuntu_laptops.yaml

```diff
@@ -68,8 +68,8 @@
     group: root
     mode: 0644
   with_items:
-    - { src: "background_16_9.png", dest: "/usr/share/backgrounds/MAG2022_lockscreen.png", }
-    - { src: "wallpaper_16_9.png", dest: "/usr/share/backgrounds/MAG2022_wallpaper.png", }
+    - { src: "background_16_9.jpeg", dest: "/usr/share/backgrounds/MAG2022_lockscreen.jpeg", }
+    - { src: "background_16_9.jpeg", dest: "/usr/share/backgrounds/MAG2022_wallpaper.jpeg", }
     - { src: "rip-and-tear.png", dest: "/usr/share/pixmaps/faces/MAG2022_face.png", }
     - { src: "rip-and-tear.png", dest: "/etc/skel/.face" }
```

Comment:
FYI: the face icons are still Doomguy, may want to come back and change that later

Reply:
@bitbyt3r